src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -268,6 +268,17 @@ public void pack_run(ChocolateyConfiguration config)
                 properties["version"] = config.Version;
             }
 
+            // Add any other properties passed to the pack command overriding any present.
+            foreach (var property in config.PackCommand.Properties)
+            {
+                this.Log().Debug(() => "{0} property '{1}': {2}".format_with(

```

Comment:
The properties passed to the command line are already dumped in Debug. Everything in the command configuration is. I figured 1) this helps me test the feature, and 2) might help someone figure out why some properties was overwritten. Using `version` above was more a means to an end to test this behavior (real-world scenario might be someone having built up the value passed to `--properties` and one value overwrites another - just like with MSBuild properties). But if you want `--version` to always win, I can certainly change that and modify the test to pass the same variable name twice.